scripts/dts/gen_defines.py

```diff
@@ -657,13 +657,24 @@ def write_vanilla_props(node):
         if prop.type in FOREACH_PROP_ELEM_TYPES:
             # DT_N_<node-id>_P_<prop-id>_FOREACH_PROP_ELEM
             macro2val[f"{macro}_FOREACH_PROP_ELEM(fn)"] = \
-                ' \\\n\t'.join(f'fn(DT_{node.z_path_id}, {prop_id}, {i})'
-                              for i in range(len(prop.val)))
+                ' \\\n\t'.join(
+                    f'fn(DT_{node.z_path_id}, {prop_id}, {i})'
+                    for i in range(len(prop.val)))
+
+            macro2val[f"{macro}_FOREACH_PROP_ELEM_SEP(fn, sep)"] = \
```

Comment:
Needs macros.bnf update (and below), no?

Reply:
right, will add them (other _FOREACH_PROP_ELEM are missing as well)